pages/overview_results_page.py

```diff
@@ -0,0 +1,592 @@
+    with gas_annual_col:
+        annual_gas_chart = create_chart_average_annual_consumption(
+            profile_annual_avgs=profile_annual_avgs,
+            energy_type="gas",
+            colour=NESTA_COLOURS[1],
+        )
+        st.altair_chart(annual_gas_chart, use_container_width=True)
+
+
+def create_household_information_expander():
+    """
+    Creates an expander for household information in the context tab.
+    """
+    st.markdown("#### 👥 Household composition")
+
+    # Age of household members
+    (
+        hc_65p_col,
+        hc_adults_child_col,
+    ) = st.columns([1, 1])
+    with hc_65p_col:
+        chart_hc_65p = plot_contextual_info(
+            data=contextual_data,
+            variable="adults_65_plus_only",
+            title="Households 65+ years old only",
+        )
+        st.altair_chart(chart_hc_65p, use_container_width=True)
+    with hc_adults_child_col:
+        chart_hc_adults_child = plot_contextual_info(
+            data=contextual_data,
+            variable="adults_and_children",
+            title="Households with adults and children",
+        )
+        st.altair_chart(chart_hc_adults_child, use_container_width=True)
+
+    # Household size and occupancy
+    occ_3more_col, occ_single_col = st.columns([1, 1])
+    with occ_3more_col:
+        chart_occupancy_3_or_more = plot_contextual_info(
+            data=contextual_data,
+            variable="3plus_occupants",
+            title="Households with 3 or more occupants",
+        )
+        st.altair_chart(chart_occupancy_3_or_more, use_container_width=True)
+    with occ_single_col:
+        chart_single_occupancy = plot_contextual_info(
+            data=contextual_data,
+            variable="single_occupancy",
+            title="Households with single occupants only",
+        )
+        st.altair_chart(chart_single_occupancy, use_container_width=True)
+
+    st.markdown("#### 💰 Working status and income")
+    # Working status
+    all_not_working_col, all_working_or_students_col, mix_col = st.columns([1, 1, 1])
+    with all_not_working_col:
+        chart_working_status = plot_contextual_info(
+            data=contextual_data,
+            variable="working_status_all_not_working",
+            title="Households where all members are not working",
+        )
+        st.altair_chart(chart_working_status, use_container_width=True)
+    with all_working_or_students_col:
+        chart_working_or_students = plot_contextual_info(
+            data=contextual_data,
+            variable="working_status_all_working_and_or_students",
+            title="Households where all members are working and/or students",
+        )
+        st.altair_chart(chart_working_or_students, use_container_width=True)
+    with mix_col:
+        chart_working_status_mix = plot_contextual_info(
+            data=contextual_data,
+            variable="working_status_mix",
+            title="Households with a mix of working and non-working members",
```

Comment:
The longer plot titles are cut-off as the column widths are quite narrow. I suggest modifying these to be multi-line via:

```suggestion
            title=["Households with a mix of working", "and non-working members"],
```